src/amber/cli/commands/new.cr

```diff
@@ -22,6 +22,11 @@ module Amber::CLI
 
       def run
         Amber::CLI.color = !options.no_color?
+        full_path_name = `pwd`.sub("\n", "") + "/name"
```

Comment:
Should this be dynamic? `"/name"`

Like `"/#{args.name}"`

Reply:
Yes. It totally should. For some reason I thought that was what I had done.